### PR TITLE
Say goodbye to `arcoe` and rwc

### PR DESCRIPTION
**Pull request by ice1000**
Basically what the title says.

### CODE REVIEW

**Comment by ice1000** [comment]
image

File: base/src/test/resources/failure/patterns/issues/issue681.aya.txt [hunk 1]
```diff
@@ -1,30 +1,12 @@
...
+  26 |   | zero, y, z => idp
+                         ^-^
 
-Error: There is no parameter for the pattern
-         zero
-       to match against, given the return type
-         Pi (x y z : Nat) -> (=) {Nat} (x + (y + z)) ((x + y) + z)
-       (and in case it's a function type, you may want to move its parameters before
-       the `:` in the signature)
+Error: The name `idp` is not defined in the current scope
```
Comment (imkiva) [inline base/src/test/resources/failure/patterns/issues/issue681.aya.txt:8]: This error is unexpected

**Pull request merged**